src/SWP/Bundle/CoreBundle/Model/FacebookInstantArticlesArticle.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+/*
+ * This file is part of the Superdesk Web Publisher Core Bundle.
+ *
+ * Copyright 2016 Sourcefabric z.ú. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2016 Sourcefabric z.ú
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\Bundle\CoreBundle\Model;
+
+use SWP\Bundle\ContentBundle\Model\ArticleInterface;
+use SWP\Component\Common\Model\TimestampableTrait;
+use SWP\Component\MultiTenancy\Model\TenantAwareInterface;
+use SWP\Component\MultiTenancy\Model\TenantAwareTrait;
+use SWP\Component\Storage\Model\PersistableInterface;
+
+class FacebookInstantArticlesArticle implements FacebookInstantArticlesArticleInterface, PersistableInterface, TenantAwareInterface
+{
+    use TenantAwareTrait, TimestampableTrait;
+
+    /**
+     * @var int
+     */
+    protected $id;
+
+    /**
+     * @var string
+     */
+    protected $submissionId;
+
+    /**
+     * @var ArticleInterface
+     */
+    protected $article;
+
+    /**
+     * @var string
+     */
+    protected $status;
+
+    /**
+     * @var array
+     */
+    protected $errors = '{}';
+
+    /**
+     * @var FacebookInstantArticlesFeedInterface
+     */
+    protected $feed;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getId(): int
+    {
+        return $this->id;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getSubmissionId(): string
+    {
+        return $this->submissionId;
+    }
+
+    /**
+     * @param string $submissionId
```

Comment:
{@inheritdoc} should be used and this method should be defined by interface if its not